docs/features/user_experience.md

```diff
@@ -4,10 +4,87 @@ The end user leverages the web interface as the primary method to engage with th
 
 ## Having a conversation with your data
 
-When you engage with IA Accelerator in the "Chat" method, the system maintains a history for your conversation and will be able to understand the context of your questions from one question to the next.
+When you use the "Chat" capabilities, the system maintains a history for your conversation and will be able to understand the context of your questions from one question to the next in a conversation flow.
```

Comment:
should we highlight the history is only at the session level and not persisted?

Reply:
absolutely! let's answer that before someone asks if we are persisting data